src/components/message/message.component.js

```diff
@@ -45,15 +45,16 @@ const Message = (props) => {
 };
 
 Message.defaultProps = {
-  messageType: 'info',
+  as: 'info',
   border: true,
   open: true,
   roundedCorners: true,
   transparent: false
 };
 
 Message.propTypes = {
-  messageType: PropTypes.oneOf(OptionsHelper.colors),
+  variant: PropTypes.oneOf(OptionsHelper.colors),
```

Comment:
Might be good to get some comments on these propTypes so they're surfaced in the storybook prop types table

Reply:
Updated :)